zib/StructureDefinitions/zib-Admission.xml

```diff
@@ -0,0 +1,280 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-Admission" />
+  <text>
+    <status value="empty" /><div xmlns="http://www.w3.org/1999/xhtml">No narrative is provided for definitional resources. A human-readable rendering can be found in the implementation guide(s) where this resource is used.</div></text>
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-Admission" />
+  <name value="ZibAdmission" />
+  <title value="zib Admission" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="url" />
+      <value value="https://www.nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="The stay of a patient or client in a healthcare facility in the context of a (partial) admission or emergency room visit. A (partial) admission is the entire or partial stay of a patient or client in a department equipped for nursing in a health care institution, for example, an inpatient ward, day care unit, emergency care, or observatory. This HCIM therefore applies to a whole admission, part of an admission, or emergency room visit in both past and present." />
+  <purpose value="This Admission resource represents the Dutch [zib (&amp;#39;Zorginformatiebouwsteen&amp;#39;, i.e. Health and Care Information Model) Admission v2.0 (2024)](https://zibs.nl/wiki/Admission-v2.0(2024EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-admission-v2.0-2024EN" />
+    <uri value="https://zibs.nl/wiki/Admission-v2.0(2024EN)" />
+    <name value="zib Admission-v2.0(2024EN)" />
+  </mapping>
+  <mapping>
+    <identity value="zib-healthprofessional-v4.0-2024EN" />
+    <uri value="https://zibs.nl/wiki/HealthProfessional-v4.0(2024EN)" />
+    <name value="zib HealthProfessional-v4.0(2024EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Encounter" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Encounter" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Encounter">
+      <path value="Encounter" />
+      <short value="Admission" />
+      <alias value="Opname" />
+      <mapping>
+        <identity value="zib-admission-v2.0-2024EN" />
+        <map value="NL-CM:15.4.1" />
+        <comment value="Admission" />
+      </mapping>
+    </element>
+    <element id="Encounter.extension:extAdmissionCommentAdmissionReason">
+      <path value="Encounter.extension" />
```

Comment:
In the mapping, this was placed on `.reasonCode.extension:commentAdmissionReason` . If you place this on the root, you could also consider using this standard extension: https://hl7.org/fhir/extensions/5.1.0/StructureDefinition-note.html
https://nictiz.atlassian.net/browse/ZIBFHIR-97

Reply:
Perhaps mapping this on `.reasonCode.text` is still a good way to go like Luud suggested in the mapping too. Discuss in the team?